src/main/java/org/sonarlint/intellij/messages/IssueStoreListener.java

```diff
@@ -25,8 +25,10 @@
 import java.util.Map;
 import org.sonarlint.intellij.issue.IssuePointer;
 
-public interface AnalysisResultsListener {
-  Topic<AnalysisResultsListener> SONARLINT_ANALYSIS_DONE_TOPIC = Topic.create("SonarLint New Issues", AnalysisResultsListener.class);
+public interface IssueStoreListener {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1214)